server/apps/immich/src/app.module.ts

```diff
@@ -16,10 +16,12 @@ import { ScheduleModule } from '@nestjs/schedule';
 import { ScheduleTasksModule } from './modules/schedule-tasks/schedule-tasks.module';
 import { DatabaseModule } from '@app/database';
 import { JobModule } from './api-v1/job/job.module';
+import { AdminConfigModule } from '@app/admin-config';
+import { ConfigModule } from "./api-v1/config/config.module";
```

Comment:
It looks like this is the same problem as immich jwt and that was renamed to ImmichJwtModule, maybe we should do the same thing here: ie `ImmichConfigModule`.